openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -146,6 +146,11 @@ jQuery(function () {
             .then((module) => module.initPatronMetadata());
     }
 
+    if (document.getElementsByClassName('imageIntro').length) {
+        import('./cover_add_manage')
```

Comment:
This requires the definition of a webpack chunk 
e.g.

```

        import(/* webpackChunkName: "book_cover_manage" */ './cover_add_manage')
            .then(module => module.initAddCoverImport());
```

Without this I'm getting the JS error   TypeError: t.initAddCoverImport is not a function

Reply:
The TypeError appears to no longer be there (checked console on Firefox), but I do have an error for archive_analytics:  

```
Uncaught ReferenceError: archive_analytics is not defined
    send_analytics_pageview http://localhost:8080/books/OL24755423M/Odyssey_book_iv/manage-covers:229
    EventListener.handleEvent* http://localhost:8080/books/OL24755423M/Odyssey_book_iv/manage-covers:228  
```

Not sure if this is important. I also got this on master, so I'll assume this is ok for now.